### ISSUE TITLE
Physical node provisioning adds unnecessary /etc/hosts entry on compute node

### ISSUE THREAD

**Issue by EmmEff** [UnivaCorporation/tortuga#79]
If the domain name of the Tortuga installer is the same as the compute node (ie. "installer.private" and "compute-01.private"), the host alias mapping for Puppet is not required. The duplicate `/etc/hosts` entry doesn't cause any harm but is not necessary in this circumstance.

Proposed solution: fix `$TORTUGA_ROOT/config/kickstart.tmpl` to detect condition under which to include `/etc/hosts` mapping.

**Comment by dtougas** [comment]
In `kickstart.tmpl` is the following section:

```
{% if installer_private_domain == domain %}
# Add a host alias for the Puppet master using the private IP address for
# hosts on the private (provisioning) network
cat >>/etc/hosts <<ENDL
$installer_private_ip $puppet_master_fqdn
ENDL
{% endif %}
```

Is the solution for this as simple as changing the `==` to a `!=` in the if statement?

**Comment by EmmEff** [comment]
I _think_ that should be "if installer_public_domain != domain" (where `domain` is the private DNS suffix).

**Comment by dtougas** [comment]
Fix in PR #82.
